### ISSUE TITLE
dataset

### ISSUE THREAD

**Issue by getiany** [palm-ml/GLLE#1]
How does the dataset become LDL_ Data set

**Comment by palm-ml** [comment]
The original dataset is transformed into LDL Dataset via label enhancement, which is essentially a pre-processing applied to the training set. 
From the algorithmic aspect, to recover the label distribution matrix $D=[d_1,d_2,...,d_n]$ from the logical label matrix $L$, GLLE assumes the parametric model
$$
d_i = W^T  \fi (x_i) + b,
$$
where $W$ is a weight matrix and $b$ is a bias vector.

Then our goal becomes to find the optimal model $W^{optimal}$ which minimizes
$$
W^{optimal} = \argmin_{W} L(W) + \lambda_1 \Omega(W) + \lambda_2 Z(W),
$$
where $L$ is a loss function, $\Omega$ is the functions to leverage the topological information of the feature space, and $Z$ is the function to leverage the correlation among labels.

From the code aspect, 'GLLE.m' is a matlab function that recovers and outputs the label distribution for the dataset.

**Comment by getiany** [comment]
thank you ,but I do not know ,At LDL_Dateset How can the logical label in dataset be calculated from the label implementation? Can you send me a code? My QQ is 605408063

**Comment by palm-ml** [comment]
You can use 'binaryzation.m' to generate the logical label in the dataset.

**Comment by palm-ml** [comment]
You can use 'binaryzation.m' to generate the logical label in the dataset.

**Comment by getiany** [comment]
thank you very much

**Comment by palm-ml** [comment]
My pleasure.
